components/home/product-cards.tsx

```
@@ -0,0 +1,24 @@
+    sold: number;
+}
+
+interface ProductCardsProps {
+  products: Product[];
+}
+
+const ProductCards = ({ products }: ProductCardsProps) => {
+    return (
+        <div>
+            {products.map((product) => (
+                <div>{product.name}</div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)